config-example.env

```diff
@@ -45,13 +45,17 @@ SQLPAD_LDAP_AUTH_ENABLED="false"
 SQLPAD_LDAP_URL=""
 
 # Base LDAP DN to search for users in.
-SQLPAD_LDAP_BASE_DN=""
+SQLPAD_LDAP_SEARCH_BASE=""
 
 # The bind user will be used to lookup information about other LDAP users.
-SQLPAD_LDAP_USERNAME=""
+SQLPAD_LDAP_BIND_DN=""
 
 # The password to bind with for the lookup user.
-SQLPAD_LDAP_PASSWORD=""
+SQLPAD_LDAP_BIND_CREDENTIALS=""
+
+# LDAP search filter, e.g. (uid={{username}}) or (sAMAccountName={{username}}). 
+# Use literal {{username}} to have the given username used in the search 
+SQLPAD_LDAP_SEARCH_FILTER=""
```

Comment:
Need to document the default value, and state that it is good enough for ActiveDirectory (if it is good enough for ActiveDirectory).

Reply:
Yeah, It is enough for ActiveDirectory and other is used enough for OpenLDAP.

Reply:
I have added the note in the comment line, please review it again.